src/NUnitFramework/framework/nunit.framework-4.5.csproj

```diff
@@ -123,6 +123,7 @@
     <Compile Include="Attributes\DatapointAttribute.cs" />
     <Compile Include="Attributes\DatapointsAttribute.cs" />
     <Compile Include="Attributes\DatapointSourceAttribute.cs" />
+    <Compile Include="Attributes\DataSourceAttribute.cs" />
```

Comment:
Even before that, I'd say. :-) We'd like to see the tests passing for all projects.